src/components/specific/models/models-manager/pdf-manager/PDFManager.scss

```diff
@@ -25,4 +25,39 @@
       text-align: center;
     }
   }
+
+  &__building-maker-btn {
+    position: absolute;
+    right: 12px;
+    top: 3px;
+  }
+  
+  &__building-maker { 
+    position: fixed;
+    top: 0%;
+    right: 0%;
+    z-index: 1000;
+    width: 423px;
+    height: 100vh;
+    box-shadow: 0 -2px 10px 0 rgba(0, 0, 0, 0.1);
+    background-color: var(--color-white);
+
+    &:deep(.bimdata-file-manager) {
```

Comment:
Nice. :ok_hand: 
Maybe we could just add a comment to explain that this is a workaround and should be temporary.